examples/launch_libmode_and_run_ingestor.py

```diff
@@ -70,11 +75,13 @@ def main():
         # Optionally, wait a bit before starting the ingestor to ensure the pipeline is ready
         time.sleep(10)
 
+        # If the pipeline is not running, exit immediately
+        if pipeline_process.poll() is not None:
+            raise RuntimeError("Error running pipeline subprocess.")
```

Comment:
This block was added because the script hangs when the pipeline generation subprocess in `start_pipeline_subprocess` fails, .e.g, when the environment variables are incorrectly set.